README.md

```diff
@@ -305,12 +311,63 @@ For an overview of the full workflow, see the [workflow diagram](./docs/workflow
    Path to taxonomy repo [taxonomy]: <ENTER>
    `taxonomy` seems to not exists or is empty. Should I clone https://github.com/instructlab/taxonomy.git for you? [y/N]: y
    Cloning https://github.com/instructlab/taxonomy.git...
-   Generating `config.yaml` in the current directory...
-   Initialization completed successfully, you're ready to start using `ilab`. Enjoy!
    ```
 
    `ilab` will use the default configuration file unless otherwise specified. You can override this behavior with the `--config` parameter for any `ilab` command.
 
+4. When prompted, provide the path to your default model. Otherwise, we will use the default of merlinite.
+
+   ```shell
+   (venv) $ ilab config init
+   Welcome to InstructLab CLI. This guide will help you set up your environment.
+   Please provide the following values to initiate the environment [press Enter for defaults]:
+   Path to taxonomy repo [taxonomy]: <ENTER>
+   `taxonomy` seems to not exists or is empty. Should I clone https://github.com/instructlab/taxonomy.git for you? [y/N]: y
+   Cloning https://github.com/instructlab/taxonomy.git...
+   Path to your model [/home/cdoern/.cache/instructlab/models/merlinite-7b-lab-Q4_K_M.gguf]: <ENTER>
+   ```
+
+5. When prompted, please choose a train profile. Train profiles are GPU specific profiles that enable accelerated training behavior. If you are on MacOS or a Linux machine without a dedicated GPU, please choose `No Profile (CPU-Only)` by hitting enter.
+
+   ```shell
+   Welcome to InstructLab CLI. This guide will help you to setup your environment.
+   Please provide the following values to initiate the environment [press Enter for defaults]:
+   Path to taxonomy repo [/home/cdoern/.local/share/instructlab/taxonomy]: 
+   Path to your model [/home/cdoern/.cache/instructlab/models/merlinite-7b-lab-Q4_K_M.gguf]: 
+   Generating `/home/cdoern/.config/instructlab/config.yaml`...
+   Please choose a train profile to use:
+   [0] No profile (CPU-only)
+   [1] A100_H100_x2.yaml
+   [2] A100_H100_x4.yaml
+   [3] A100_H100_x8.yaml
+   [4] L40_x4.yaml
+   [5] L40_x8.yaml
+   [6] L4_x8.yaml
+   Enter the number of your choice [hit enter for the default CPU-only profile] [0]:  
+   Using default CPU-only train profile.
+   Initialization completed successfully, you're ready to start using `ilab`. Enjoy!
+   ```
+
+   The GPU profiles are listed by GPU type and amount. If you happen to have a GPU configuration with a similar amount of vram as any of the above configs, feel free to try them out!
```

Comment:
GPU type and amount -> number? (I think the former is for "mass nouns").

Reply:
VRAM?

Reply:
`above configs` -> maybe `profiles`?

Reply:
did all of these!